src/actions.js

```diff
@@ -87,8 +87,44 @@ export function fetchOrganisations() {
       .then(responseObj => responseObj.json())
       .then(responseData => {
         const data = responseData.results;
+        const allOrganisations = data.map(organisation => {
+          //use organisation uuid without dashes only
```

Comment:
This is bad, IDs should be black boxes. Why was that choice made again?

Reply:
made ticket:
https://nelen-schuurmans.atlassian.net/browse/FRNT-555